storage/innobase/include/buf0buf.h

```diff
@@ -2090,6 +2096,14 @@ struct buf_pool_t{
 
 	/** @name General fields */
 	/* @{ */
+	BufListMutex chunks_mutex;    /*!< protects (de)allocation of chunks:
+									- changes to chunks, n_chunks are performed
+									while holding this latch,
+									- reading buf_pool_should_madvise requires
+									holding this latch for any buf_pool_t
+									- writing to buf_pool_should_madvise requires
+									holding these latches for all buf_pool_t-s
```

Comment:
is this properly aligned? cannot makeout from github page

Reply:
no, it is not. Thanks for the heads up